muxink/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "muxink"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1.0.57"
```

Comment:
Can we get rid of this dependency? It always seemed to me that it's a good one if you write a CLI tools and don't bother about handling errors properly. `muxink` is a library and probably a very-well written so it does (🤞 ) return all results in a typed fashion?

Reply:
Good catch. It's actually not in use anymore, it was used in earlier versions.

Reply:
Fixed in 2c98125a0